File: eTeatar/eTeatar.Model/Uplata.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Uplata
    {
        public int UplataId { get; set; }

        public decimal Iznos { get; set; }

        public string TransakcijaId { get; set; }

        public string NacinPlacanja { get; set; }

        public string Status { get; set; }

        public DateTime Datum { get; set; }

        public int KorisnikId { get; set; }

        public virtual ICollection<StavkaUplate> StavkaUplates { get; set; } = new List<StavkaUplate>();
    }
}

File: eTeatar/UI/eteatar_desktop/lib/models/uplata.dart
import 'package:json_annotation/json_annotation.dart';

part 'uplata.g.dart';

@JsonSerializable()
class Uplata {
  int? uplataId;
  double? iznos;
  DateTime? datum;
  int? korisnikId;
  String? transakcijaId;
  String? nacinPlacanja;  
  String? status;

  Uplata(
    this.uplataId,
    this.iznos
  );

  factory Uplata.fromJson(Map<String, dynamic> json) => _$UplataFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$UplataToJson(this);
}